docs/reference/schema.md

```diff
@@ -1,7 +1,235 @@
+The tables below describe each of the fields and objects in OC4IDS. To see how they fit together, consult the [schema browser](browser.md).
 
-<script src="../../_static/docson/widget.js" data-schema="../../project-schema.json"></script>
+## Project
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :include:
+    :collapse: period,assetLifetime,sector,additionalClassifications,locations,budget/amount,budget/budgetBreakdown,parties,documents,contractingProcesses
+```
+
+## ContractingProcess
+
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :include:
+    :pointer: /definitions/ContractingProcess
+    :collapse: releases,summary
+
+```
+
+## ContractingProcessSummary
+
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :include:
+    :pointer: /definitions/ContractingProcessSummary
+    :collapse: documents
+
+```
+
+## LinkedRelease
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :include:
+    :pointer: /definitions/LinkedRelease
+
+```
+
+
+## Components
+
+### Classification
+
+A classification consists of an identifier for the codelist (the `scheme`) and a code from that codelist (the `id`), and then a human-readable label for the classification (the `description`).
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :pointer: /definitions/Classification
+    :include:
+    :collapse:
+
+```
+
+For example:
+
+```json
+{
+    "scheme":"COFOG",
+    "id":"05.2",
+    "description":"Waste water management"
+}
+```
+
+### Organization
+
+For each organization, provide as much structured data as you can.
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :pointer: /definitions/Organization
+    :collapse: identifier,additionalIdentifiers,address,contactPoint
+
+```
+
+### OrganizationReference
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :pointer: /definitions/OrganizationReference
+    :include:
+    :collapse:
+
+```
+
+### Identifier
+
+Use of stable official organisation identifiers can help join up data between systems.
+
+Organization identifiers should be constructed by collecting an official company (or government body) registration number for the organisation, and then finding the [org-id.guide list code](http://www.org-id.guide) for the list this identifier is taken from to use in the `scheme` field.
+
+For example, if identifying a company in Colombia, look up its identifier in the [Unified Commercial and Social Registry](http://org-id.guide/list/CO-RUE) and use the list code `CO-RUE`.
+
+```eval_rst
+
+.. jsonschema:: ../../build/current_lang/project-schema.json
+    :pointer: /definitions/Identifier
+    :include:
+    :collapse:
+
+```
+
+
+### Locations
```

Comment:
@duncandewhurst  I just noticed as part of this PR: The "Components" should be sorted alphabetically, and `### Locations` should be `### Location`.